.github/workflows/automate-stale.yml

```diff
@@ -0,0 +1,42 @@
+name: 'Automation - Stale issues and PRs'
+
+on:
+  schedule:
+    - cron: '0 7 * * 1-5'
+
+env:
+  DAYS_BEFORE_STALE: 60
+  DAYS_BEFORE_CLOSE: 7
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Run Stale Bot
+        id: stale-bot
+        uses: actions/stale@v9
+        with:
+          # General settings
+          days-before-stale: ${{ env.DAYS_BEFORE_STALE }}
+          days-before-close: ${{ env.DAYS_BEFORE_CLOSE }}
+          operations-per-run: 3000
+          exempt-all-milestones: true
+          enable-statistics: true # This is only useful if secret ACTIONS_STEP_DEBUG=true
+          start-date: '2025-01-17'
+          remove-stale-when-updated: true
+
+          # PR settings
+          stale-pr-label: stale
+          stale-pr-message: |
+            Hi! This pull request hasn't had any activity for a while, so I am
+            marking it as stale. It will close in ${{ env.DAYS_BEFORE_CLOSE }}
+            days unless something changes. Thanks for contributing!
```

Comment:
```suggestion
            Hi! This pull request hasn't had any activity for a while, so I am
            marking it as stale. It will close in ${{ env.DAYS_BEFORE_CLOSE }}
            days if it is not updated. Thanks for contributing!
```